test/parallel/test-stream-pipe-error-handling.js

```diff
@@ -108,3 +108,14 @@ const Stream = require('stream').Stream;
   w.removeListener('error', () => {});
   removed = true;
 }
+
+{
+  const destination = new PassThrough();
+  destination.once('error', common.mustCall());
```

Comment:
Can you please verify that the error is the one passed to `destination.destroy()`?

Reply:
Fixed